hugegraph-core/src/main/java/org/apache/hugegraph/structure/HugeProperty.java

```diff
@@ -39,11 +39,10 @@
     public HugeProperty(HugeElement owner, PropertyKey pkey, V value) {
         E.checkArgument(owner != null, "Property owner can't be null");
         E.checkArgument(pkey != null, "Property key can't be null");
-        E.checkArgument(value != null, "Property value can't be null");
 
         this.owner = owner;
         this.pkey = pkey;
-        this.value = pkey.validValueOrThrow(value);
+        this.value = pkey.validValue(value);
```

Comment:
note also need to update `HugeEdge.property(String key, V value)` to add null value check (the same as `HugeVertex.property(String key, V value)`): 
```java
        if (value == null) {
            this.removeProperty(propertyKey.id());
            return EmptyProperty.instance();
        }
```